src/tool/confiscan.sh

```diff
@@ -116,6 +116,73 @@ else
     mkdir -p "${output_dir}"
 fi
 
+######################################
+# Disks, partitions and mount points #
+######################################
+
+info "Disk info:"
+printf 'Disk,Size,Bytes,Sectors\n' > "${output_dir}/disk_info.csv"
+fdisk -l | \
+    grep 'Disk /' | \
+    sed -e 's/Disk //;s/:/,/;s/GiB//;s/bytes//;s/sectors//;s/ //g' >> \
+        "${output_dir}/disk_info.csv"
+column -s, -t "${output_dir}/disk_info.csv"
+
+info "Partitions:"
+printf 'Device,Start,End,Sectors,Size,Type\n' > "${output_dir}/partitions.csv"
+while IFS=, read -r line; do
+    # My RegEx skills aren't that impressive to do this in one sed command.
+    # So I'm using multiple commands to get the values.
+    # It's not the most optimal way, but hey it works. ;)
+    disk=$(printf '%s' "${line}" | cut -d',' -f1)
+    start=$(printf '%s' "${line}" | cut -d',' -f2)
+    end=$(printf '%s' "${line}" | cut -d',' -f3)
+    sectors=$(printf '%s' "${line}" | cut -d',' -f4)
+    size=$(printf '%s' "${line}" | cut -d',' -f5)
+    type=$(printf '%s' "${line}" | cut -d',' -f6- | sed 's/,/ /g')
+
+    printf '%s,%s,%s,%s,%s,%s\n' \
+        "${disk}" \
+        "${start}" \
+        "${end}" \
+        "${sectors}" \
+        "${size}" \
+        "${type}" >> \
+        "${output_dir}/partitions.csv"
+done < <(fdisk -l | grep -E '^/dev/' | tr -s '[:space:]' | sed 's/ /,/g')
```

Comment:
@bdeferme do you know a better way of getting all these values? For example with one sed command? 😉

Reply:
Refactored in commit: https://github.com/AntonVanAssche/hogent-bachelorproef/pull/18/commits/90a5eef62e40bd7cdb10ce7e9d93e44024c6c88e